db/migrate/20200809160820_create_tickets.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateTickets < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tickets do |t|
+      t.references :contact, null: true, foreign_key: true
```

Comment:
all tickets should be owned by a contact aka someone should be redeeming it so null: false

Reply:
👍